docs/logs/customizing-the-sdk/README.md

```diff
@@ -26,7 +26,11 @@ See [Program.cs](Program.cs) for an example.
 
 ### IncludeFormattedMessage
 
-TODO
+`IncludeFormattedMessage` will invoke a formatter with the signature
+`Func<TState, Exception, string>` to set the value of `LogRecord.FormattedMessage`.
+The default message formatter will only return the value of the `object TState`.
+
+See [Program.cs](Program.cs) for an example of providing a custom formatter.
```

Comment:
do we need to document custom formatter here? the setting only controls whether LogRecord includes/not includes FormattedMessage.